MIGRATION.md

```diff
@@ -0,0 +1,52 @@
+# Migration Guides
+
+Below are instructions for migrating from one version of dc-static-blog-next-js to another.
+
+## `1.x.x` to `2.x.x`
+
+### Prerequisites
+
+The following features need to be enabled on your hub before this migration can occur:
+
+- Search Indexes
+- Content Delivery 2
+
+### Create and configure index
+
+Ensure that you have setup and configured a search index by following [README.md](README.md) sections `Creating a production search index for your published blog-posts`,
+`Configure your index`, `Add sort options` and `Configure webhook custom payload`.
+
+### Setup and update content
+
+- Create a new `blog` content type - see `Creating a Blog content item` in the [README.md](README.md)
+
+- In 2.x.x, the content property `slug` is no longer used to build blog pages and had been replaced by the content items Delivery Key. To maintain existing blog pages you will need to copy the `slug` for each blog post and save it as the content items Delivery Key. This needs to be done for each content item before updating the content types.
+
```

Comment:
Missing the step: Re-publish the content items to backfill the search index.

This should be done before the syncing on the content items, because syncing content type before you re-publish your blog posts will break your existing v1 blog.